### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/client-platforms/aurelia/01-login.md [hunk 1]
```diff
@@ -12,9 +12,10 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'auth0-aurelia',
```
Comment (mpaktiti) [inline articles/client-platforms/aurelia/01-login.md:15]: @mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
  Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this and also rebase this PR to resolve conflicts? Thanks.

**Comment by mehreencs87** [comment]
@mpaktiti check the latest commit.

**Comment by mpaktiti** [comment]
@mehreencs87 The file has been edited and there are conflicts in this PR, can you please rebase? Thanks.

**Comment by mehreencs87** [comment]
@mpaktiti updated here https://github.com/auth0/docs/pull/2046

**Pull request closed without merging**